public/app/features/alerting/unified/api/alertRuleApi.ts

```diff
@@ -206,6 +207,10 @@ export const alertRuleApi = alertingApi.injectEndpoints({
       providesTags: ['CombinedAlertRule'],
     }),
 
+    getGrafanaAlertRule: build.query<RulerGrafanaRuleDTO, { uid: string; ruler?: string }>({
+      query: ({ uid, ruler = GRAFANA_RULES_SOURCE_NAME }) => ({ url: `/api/ruler/${ruler}/api/v1/rule/${uid}` }),
```

Comment:
Finally 🚀

Reply:
one nit; can we choose a more generic name here? It's _currently_ only for Grafana managed rules but no reason why we couldn't add this in to Mimir either.

Reply:
Are you okay with `getAlertRule`? To keep it simple - the parameters will give the rest of the context of what we're fetching, and at it's core we're getting an alert rule

Reply:
> no reason why we couldn't add this in to Mimir either

I think the main reason is the lack of unique identifiers in Mimir

Reply:
Should we remove the `ruler` parameter? The endpoint doesn't exists for any other than Grafana ruler